arrocke/gloss-translation
# feat: translator notes on words

## What has changed



- Filled in the translator notes tab of the translation sidebar.
- Added `PATCH /languages/{code}/words/{wordId}/note` for updating the translator note for a word
- Added `GET /languages/{code}/verses/{verseId}/notes` for getting the translator notes for a verse
- Renamed `GET /languages/{code}/verses/{verseId}` to `GET /languages/{code}/verses/{verseId}/glosses`
    - Funny enough, this is how it was already listed in `spec/index.yml`
    - I also updated the spec file for this route, it was quite a bit out of date.

## Connected Issues



#310

## QA Steps



1. Open the sidebar for a word
2. Go to the notes tab
3. Click "Edit"
4. Enter some notes
5. Click "Save"
6. Refresh the page
7. Go to the notes tab for the same word
    - The previously saved note should persist
    - The author and last updated fields should be accurate

## Post-Deployment
